cost-analyzer/values.yaml

```diff
@@ -2892,109 +2892,131 @@ costEventsAudit:
+  # azureSubscriptionID: 0bd50fdf-c923-4e1e-850c-196dd3dcc5d3
+  # azureClientID: f2ef6f7d-71fb-47c8-b766-8d63a19db017
+  # azureTenantID: 72faf3ff-7a3f-4597-b0d9-7b0b201bb23a
+  # azureClientPassword: fake key # Only use if your values.yaml are stored encrypted. Otherwise provide an existing secret via serviceKeySecretName
+  # azureOfferDurableID: "MS-AZR-0003p"
+  # discount: "" # percentage discount applied to compute
+  # negotiatedDiscount: "" # custom negotiated cloud provider discount
+  # defaultIdle: false
+  # serviceKeySecretName: "" # Use an existing AWS or Azure secret with format as in aws-service-key-secret.yaml or azure-service-key-secret.yaml. Leave blank if using createServiceKeySecret
+  # createServiceKeySecret: true # Creates a secret representing your cloud service key based on data in values.yaml. If you are storing unencrypted values, add a secret manually
+  # sharedNamespaces: "" # namespaces with shared workloads, example value: "kube-system\,ingress-nginx\,kubecost\,monitoring"
+  # sharedOverhead: "" # value representing a fixed external cost per month to be distributed among aggregations.
+  # shareTenancyCosts: true # enable or disable sharing costs such as cluster management fees (defaults to "true" on Settings page)
+  # metricsConfigs: # configuration for metrics emitted by Kubecost
+  #   disabledMetrics: [] # list of metrics that Kubecost will not emit. Note that disabling metrics can lead to unexpected behavior in the cost-model.
+  # productKey: # apply business or enterprise product license
+  #   key: ""
+  #   enabled: false
+  #   secretname: productkeysecret # create a secret out of a file named productkey.json of format { "key": "kc-b1325234" }. If the secretname is specified, a configmap with the key will not be created
+  #   mountPath: "/some/custom/path/productkey.json" # (use instead of secretname) declare the path at which the product key file is mounted (eg. by a secrets provisioner). The file must be of format { "key": "kc-b1325234" }
+
+  ## Specify an existing Kubernetes Secret holding the cloud integration information. This Secret must contain
+  ## a key with name `cloud-integration.json` and the contents must be in a specific format. It is expected
+  ## to exist in the release Namespace. This is mutually exclusive with cloudIntegrationJSON where only one must be defined.
+  # cloudIntegrationSecret: "cloud-integration"
+
+  ## Specify the cloud integration information in JSON form if pointing to an existing Secret is not desired or you'd rather
+  ## define the cloud integration information directly in the values file. This will result in a new Secret being created
+  ## named `cloud-integration` in the release Namespace. It is mutually exclusive with the cloudIntegrationSecret where only one must be defined.
+  # cloudIntegrationJSON: |-
+  #   {
```

Comment:
would it make sense if we also added example configs for GCP, and Azure?

Reply:
```
"azure": [
	{
		"azureSubscriptionID": "my-subscription-id",
		"azureStorageAccount": "my-storage-account",
		"azureStorageAccessKey": "my-storage-access-key",
		"azureStorageContainer": "my-storage-container"
	}
],
"gcp": [
	{
		"projectID": "my-project-id",
		"billingDataDataset": "detailedbilling.my-billing-dataset",
		"key": {
			"type": "service_account",
			"project_id": "my-project-id",
			"private_key_id": "my-private-key-id",
			"private_key": "my-pem-encoded-private-key",
			"client_email": "my-service-account-name@my-project-id.iam.gserviceaccount.com",
			"client_id": "my-client-id",
			"auth_uri": "auth-uri",
			"token_uri": "token-uri",
			"auth_provider_x509_cert_url": "my-x509-provider-cert",
			"client_x509_cert_url": "my-x509-cert-url"
		}
	}
],
```

Reply:
Thank you. How about the AWS values? Are those correct and also complete?

Reply:
yes they look good so I omitted those

Reply:
Awesome, thanks. Just pushed those up so have a look.